app/services/private_interfaces/destroyer.rb

```diff
@@ -2,12 +2,26 @@ module PrivateInterfaces
   class Destroyer
     include Serviceable
 
+    def call(environment)
+      @environment = environment
+      @service = "#{environment.app_id}-#{environment.name}-private"
+      uninstall(@service) if exists?(@service)
+    end
+
+    private
+
+    attr_reader :environment
```

Comment:
wondering if we can also add `:service` to the list of `attr_reader`s. So then on Line 8, we can call `service` without the `@` sign, just as `environment` is called on Line 7!

Reply:
I tried something like this but got tons of errors... apparently service is not passed to this class
=P